### PR TITLE
AML-2021, AML-2028, AML-2109 View/Approve/Reject/Toggle transfer connection invitations

### PR DESCRIPTION
**Pull request by elliotritchie**

### CODE REVIEW

File: src/SFA.DAS.EAS.Web/ViewModels/TransferConnectionInvitations/ReceiveTransferConnectionInvitationViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        [Required]
+        public RejectTransferConnectionInvitationCommand RejectTransferConnectionInvitationCommand { get; set; }
+
+        [Required(ErrorMessage = "Option required")]
+        [RegularExpression("Approve|Reject", ErrorMessage = "Option required")]
+        public string Choice { get; set; }
+
+        public TransferConnectionInvitationDto TransferConnectionInvitation { get; set; }
+
+        public override void Map(ApproveTransferConnectionInvitationCommand message1, RejectTransferConnectionInvitationCommand message2)
```
Comment (robinoconnor) [inline src/SFA.DAS.EAS.Web/ViewModels/TransferConnectionInvitations/ReceiveTransferConnectionInvitationViewModel.cs]: rename message1 and message2 to more meaningful names

File: src/SFA.DAS.EAS.Employer_Account.Database/Tables/TransferConnectionInvitation.sql [hunk 2]
```diff
@@ -4,9 +4,11 @@
 	[SenderUserId] BIGINT NOT NULL,
 	[SenderAccountId] BIGINT NOT NULL,
 	[ReceiverAccountId] BIGINT NOT NULL,
-	[Status] TINYINT NOT NULL,
+	[ApproverUserId] BIGINT NULL,
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Employer_Account.Database/Tables/TransferConnectionInvitation.sql]: Yeah sure, `ActionerId` sound ok?

File: src/SFA.DAS.EAS.Web/App_Start/DatabaseConfig.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Data.Entity;
+using SFA.DAS.EAS.Application.Data;
+
+namespace SFA.DAS.EAS.Web
+{
+    public class DatabaseConfig
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/App_Start/DatabaseConfig.cs]: Yeah both can be initialized there.

File: src/SFA.DAS.EAS.Web/Authentication/CurrentUserService.cs [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
+            var isAuthenticated = _owinService.IsUserAuthenticated();
 
             if (!isAuthenticated)
             {
                 return null;
             }
 
-            var externalUserId = _owinWrapper.GetClaimValue(ControllerConstants.SubClaimKeyName);
-            var email = _owinWrapper.GetClaimValue(ControllerConstants.EmailClaimKeyName);
+            var externalId = Guid.Parse(_owinService.GetClaimValue(ControllerConstants.SubClaimKeyName));
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/Authentication/CurrentUserService.cs]: Yeah I'm happy with throwing an exception here.

**Pull request merged**